<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
	background: QLinearGradient( x1: 0, y1:0, x2:0, y2:1, stop:0.0 silver,
	stop: 0.2 #a8a8a8, stop: 1 #727272);
	border: 1px solid #3A3939;
	width: 12px;
	margin: 0 -4px;
	border-radius:2px;
}

QAbstractSpinBox {
	padding-top: 2px;
	padding-bottom: 2px;
	border: 1px solid darkgray;

	border-radius: 2px;
	min-width: 50px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>571</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:vertical {
	background: QLinearGradient( x1: 0, y1:0, x2:0, y2:1, stop:0.0 silver,
	stop: 0.2 #a8a8a8, stop: 1 #727272);
	border: 1px solid #3A3939;
	width: 12px;
	margin: 0 -4px;
	border-radius:2px;
}

QAbstractSpinBox {
	padding-top: 2px;
	padding-bottom: 2px;
	border: 1px solid darkgray;

	border-radius: 2px;
	min-width: 50px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter Video URL Here.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Title:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Duration:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonGo">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Choose Quality</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Quality</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Rating:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Quality</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>471</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Views:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>501</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblMsg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>499</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>lblMsg</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter Video URL Here.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Videos Count:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Quality</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>380</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxq">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Quality</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>501</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblMsg2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>iblMsg</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="text">
     <string>Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="text">
     <string>About Us</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
